.github/workflows/universalnrp-test.yml

```diff
@@ -190,7 +190,8 @@ jobs:
         uses: ./.github/actions/check-size
         with:
           package: '${{ steps.normalize.outputs.path }}'
-          limit: 350
+          # TODO: Adjust size limit as needed
```

Comment:
># TODO: Adjust size limit as needed [](http://example.com/codeflow?start=10&length=35)

How would we know when this limit would be in need to be increased.... or is too large and we need it decreased?

And speaking of this, what is this limit for and what are the units in? Can we have this described in a comment instead of the bare TODO one?

Reply:
I will remove the TODO but ive already checked the release package size and the largest, being the complianceengine is ~4800KB so ive left 5000KB as the ceiling.